src/index.html

```diff
@@ -5,8 +5,376 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo-link">
+        <a href="#" class="header__logo">
+          <img src="./images/logo.png" alt="Logo Moyo">
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__items">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="card" data-qa="card">
+          <div class="card__image-block">
+            <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 inches"
+          >
```

Comment:
Fix formatting in all code.
![image](https://github.com/mate-academy/layout_catalog/assets/54860641/f8a526b0-bb99-4ea3-8ba9-31b993b9cd7e)